app/javascript/mastodon/locales/en.json

```diff
@@ -271,6 +272,8 @@
   "filter_modal.select_filter.subtitle": "Use an existing category or create a new one",
   "filter_modal.select_filter.title": "Filter this post",
   "filter_modal.title.status": "Filter a post",
+  "filtered_notifications_banner.pending_requests": "Notifications from {count, plural, =0 {no} one {one person} other {# people}} you may know",
```

Comment:
when count is 0, source text says "Notifications from no you may know", which makes no sense

Reply:
That should've been "no one", but also, the banner is hidden when the count is 0.